tests/unit/introspection.tcl

```diff
@@ -670,6 +670,7 @@ start_server {tags {"introspection"}} {
         set r1 [redis_client]
         assert_equal [$r1 ping] "PONG"
         $r1 close
+        #close $sockfd
```

Comment:
mistake?

Reply:
yes. i tested that each of these changes on it's own solves the problem.
forgot to undo this comment.